src/depackers/stb_vorbis.c

```diff
@@ -4053,6 +4126,12 @@ static int start_decoder(vorb *f)
             }
          }
       }
+
+      /* Sanity check */
+      if (r->classbook >= f->codebook_count) {
+         return error(f, VORBIS_invalid_setup);
```

Comment:
It looks like this same check has been added in mainstream, but above around line 4107